src/plugins/discover/public/application/components/sidebar/discover_field.tsx

```diff
@@ -289,7 +301,8 @@ export function DiscoverField({
   const isRuntimeField = Boolean(indexPattern.getFieldByName(field.name)?.runtimeField);
   const isUnknownField = field.type === 'unknown' || field.type === 'unknown_selected';
   const canEditField = onEditField && (!isUnknownField || isRuntimeField);
-  const displayNameGrow = canEditField ? 9 : 10;
```

Comment:
can you explain the grow stuff? It seems like this would be way simpler by just using `grow={true}` on the display name and `grow={false}` on the buttons (which is what Lens is doing: https://github.com/elastic/kibana/blob/c218ce292af1cc90ce619fbc4f1b7459e3255d4b/x-pack/plugins/lens/public/indexpattern_datasource/field_item.tsx#L334 )

Reply:
I spent almost a day trying to get this to align nicely 😅 It was ugly. Will try it once again